app/src/main/java/org/sagebionetworks/research/mpower/tracking/fragment/SelectionFragment.java

```diff
@@ -0,0 +1,138 @@
+    private TrackingActiveTaskViewModel<SimpleTrackingItemConfig, SimpleTrackingItemLog> viewModel;
+
+    @NonNull
+    public static SelectionFragment newInstance(@NonNull StepView step) {
+        SelectionFragment fragment = new SelectionFragment();
+        Bundle args = SelectionFragment.createArguments(step);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onAttach(Context context) {
+        AndroidSupportInjection.inject(this);
+        super.onAttach(context);
+    }
+
+    @NonNull
+    public static Bundle createArguments(@NonNull StepView step) {
+        checkNotNull(step);
+
+        Bundle bundle = new Bundle();
+        bundle.putSerializable(ARGUMENT_STEP_VIEW, step);
+        return bundle;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        TrackingStepView stepView = null;
+        if (savedInstanceState == null) {
+            Bundle arguments = getArguments();
+            if (arguments != null) {
+                // noinspection unchecked
+                stepView = (TrackingStepView)this.getArguments().getSerializable(ARGUMENT_STEP_VIEW);
+            }
+        } else {
+            // noinspection unchecked
+            stepView = (TrackingStepView)savedInstanceState.getSerializable(ARGUMENT_STEP_VIEW);
+        }
+
+        checkState(stepView != null, "stepView cannot be null");
+        this.stepView = stepView;
+        this.viewModel = ViewModelProviders.of(this, this.trackingActiveTaskViewModelFactory.create(this.stepView))
+                .get(SimpleTrackingActiveTaskViewModel.class);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View result = inflater.inflate(R.layout.mpower2_selection_step, container, false);
+        this.unbinder = ButterKnife.bind(this, result);
+        OnApplyWindowInsetsListener topInsetListener = SystemWindowHelper.getOnApplyWindowInsetsListener(Direction.TOP);
+        ViewCompat.setOnApplyWindowInsetsListener(this.cancelButton, topInsetListener);
+        this.title.setText(this.stepView.getSelectionInfo().getTitle());
+        this.detail.setText(this.stepView.getSelectionInfo().getDetail());
+        LinearLayoutManager manager = new LinearLayoutManager(this.recyclerView.getContext());
+        this.recyclerView.setLayoutManager(manager);
+        DividerItemDecoration itemDecoration = new DividerItemDecoration(this.recyclerView.getContext(), manager.getOrientation());
+        Drawable drawable = this.getContext().getResources().getDrawable(R.drawable.form_step_divider);
+        itemDecoration.setDrawable(drawable);
+        this.recyclerView.addItemDecoration(itemDecoration);
+        TrackingItemAdapter adapter = new TrackingItemAdapter(this.viewModel.getAvailableElements().getValue(), this.viewModel);
```

Comment:
Does this work? I don't see an observer for the LiveData. 

Something like this in onCreate
viewModel.getAvailableElements().observe(this, it -> {
            adapter.loadItems(it);
            adapter.notifyDataSetChanged();
        });

Using a method instead of lambda would make it easier to UI test